koku/masu/processor/report_parquet_processor_base.py

```diff
@@ -124,18 +124,29 @@ def _generate_create_table_sql(self):
             sql += f"{norm_col} {col_type}"
             if idx < (len(parquet_columns) - 1):
                 sql += ","
-        sql += ",source varchar, year varchar, month varchar"
+        if partition_map:
+            # Add the specified partition columns
+            sql += ", "
+            sql += ",".join([f"{item[0]} {item[1]} " for item in list(partition_map.items())])
```

Comment:
If `partion_map` was an Enum, it would look something like this:
```suggestion
            sql += ",".join([f"{item.name} {item.value} " for item in list(partition_map])
```

Reply:
The comment above is related to this one about Enum's likely not being the best choice for this scenario.